integration_tests/test_ibc.py

```diff
@@ -0,0 +1,164 @@
+import pytest
+import time
+import os
+import json
+from .conftest import setup_cronos2, setup_chainmain, setup_hermes
+from web3 import Web3
+import json
+import sys
+
+
+@pytest.fixture(scope="module")
+def cronos(tmp_path_factory):
+    print(f'cronos tmp folder= {tmp_path_factory}')
+    yield from setup_cronos2(tmp_path_factory.mktemp("cronos"), 26700)
+
+
+@pytest.fixture(scope="module")
+def chainmain(tmp_path_factory):
+    print(f'chainmain tmp folder= {tmp_path_factory}')
+    # "start-cronos"
+    yield from setup_chainmain(tmp_path_factory.mktemp("chainmain"), 26800)
+
+
+@pytest.fixture(scope="module")
+def hermes(tmp_path_factory):
+    print("hermes waiting for chains booting up")
+    time.sleep(20)
+    print(f'hermes tmp folder= {tmp_path_factory}')
+    yield from setup_hermes(tmp_path_factory.mktemp("hermes"), 26900)
+
+
+def getBalance(chain, addr, denom):
+    output = chain.cosmos_cli(0).raw(
+        "query",
+        "bank",
+        "balances",
+        addr,
+        node=chain.node_rpc(0),
+        output="json",
+    )
+    c = json.loads(output.decode())
+    d = json.dumps(c, indent=4)
+    print(d)
+    coins = c["balances"]
+    for coin in coins:
+        if coin["denom"] == denom:
+            value = int(coin["amount"])
+            return value
+    return 0
+
+
+def test_ibc(cronos, chainmain, hermes):
+    # wait for hermes
+    time.sleep(20)
+    while True:
+        print(f"test ibc config={hermes.configpath}")
+        MYIBC0 = "chainmain-1"
+        MYIBC1 = "cronos_777-1"
+        MYCHANNEL = "channel-0"
+        MYCONFIG = hermes.configpath
+        # signer21
+        RECEIVER= "cro1apdh4yc2lnpephevc6lmpvkyv6s5cjh652n6e4"        
+        SRCAMOUNT = 5
+        SRCDENOM = "basetcro"
+        DSTDENOM ="ibc/6B5A664BF0AF4F71B2F0BAA33141E2F1321242FBD5D19762F541EC971ACB0865"        
+        # dstchainid srcchainid srcportid srchannelid
+        # chainmain-1 <- cronos_777-1
+        cmd = f'hermes -c {MYCONFIG} tx raw ft-transfer {MYIBC0} {MYIBC1} transfer {MYCHANNEL} {SRCAMOUNT} -o 1000 -n 1 -d {SRCDENOM} -r {RECEIVER} -k testkey'
+        print(f'hermes config= {MYCONFIG}')
+        stream = os.popen(cmd)
+        output = stream.read()
+        print(output)
+        dstaddr = f"{RECEIVER}"
+        olddstbalance = getBalance(chainmain, dstaddr, DSTDENOM)
+        print(f"old balance={olddstbalance} denom={DSTDENOM}")
+        time.sleep(5)
+        newdstbalance = getBalance(chainmain, dstaddr, DSTDENOM)
+        print(f"new balance={newdstbalance} denom={DSTDENOM}")
+        assert olddstbalance + SRCAMOUNT == newdstbalance
```

Comment:
does it work?

Current implementation, `newdstbalance = olddstbalance + SRCAMOUNT * 10^10 ` because of conversion module